cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/builder/Fluent.kt

```diff
@@ -50,6 +67,86 @@ fun LanguageFrontend.translationUnit(
 
     scopeManager.resetToGlobal(node)
     init(node)
+    this@TranslationResult.components.firstOrNull()?.translationUnits?.add(node)
+
+    return node
+}
+
+/**
+ * Creates a new [TranslationUnitDeclaration] in the Fluent Node DSL with the given [name]. The
+ * declaration will be set to the [ScopeManager.globalScope]. The [init] block can be used to create
+ * further sub-nodes as well as configuring the created node itself.
+ */
+fun LanguageFrontend.translationUnit(
```

Comment:
Why do we have this twice now?

Reply:
One of them requires the `TranslationResult` as a context whereas this one doesn't. It's related to `build`vs `buildTR` from above. If we always return the translation result in `build`, we can get rid of this method.